modules/spo-container-profile-instances.adoc

```diff
@@ -43,6 +44,11 @@ spec:
       app: sp-record
 ----
 
+. Label the namespace
++
+[source,terminal]
+$ oc label ns my-namespace
```

Comment:
Should be a full command like
`oc label ns my-namespace security.openshift.io/scc.podSecurityLabelSync=false pod-security.kubernetes.io/enforce=privileged --overwrite=true`
Or the pod will be failed to create due to PSA issue.

Reply:
Thank you, this is done!